h5py/tests/test_file.py

```diff
@@ -762,6 +780,15 @@ def test_mpio(self, mpi_file_name):
             assert f
             assert f.driver == 'mpio'
 
+    def test_mpio_append(self, mpi_file_name):
+        """ Testing creation of file with append """
+        from mpi4py import MPI
+
+        if os.path.exists(mpi_file_name): os.remove(mpi_file_name)
```

Comment:
I believe `mpi_file_name` is guaranteed not to exist when the test starts (@aragilar ?). And if not, I suspect that this check doesn't work reliably in MPI - if >1 process realises the file exists, they'll race to delete it.

Reply:
Ok I checked and it is unique. I will remove that.